python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -241,3 +241,19 @@ def test_groupby_string_index_name(myindex):
     gdf = ddf.groupby("index").agg({"data": "count"})
 
     assert gdf.compute().index.name == gdf.index.name
+
+
+def test_groupby_split_out_multiindex():
+    import cudf
+    import dask.dataframe as dd
+    import dask_cudf
+    import numpy as np
+
+    df = cudf.DataFrame(
+        {"a": np.arange(100), "b": np.arange(100), "c": np.random.random(100)}
+    )
+    ddf = dask_cudf.from_cudf(df, 5)
+    pddf = dd.from_pandas(df.to_pandas(), 5)
+    gr = ddf.groupby(["a", "b"]).agg({"c": "count"}, split_out=2)
```

Comment:
Thanks @thomcom for addressing this on such short notice! To guard against future issues can we also add test cases where the agg is being called like :
`ddf.groupby(["a", "b"]).c.count(split_out=2)`
`ddf.groupby(["a", "b"]).count(split_out=2)` 
`ddf.groupby(["a", "b"]).agg({"c": ["count", "sum"]}, split_out=2)`

Dask takes a slightly different codepath based on how the aggregation is called and there have been other issues where the problem presents only when called a specific way.

Reply:
Coming up shortlly!